app/models/user-app-meta/intro-content.ts

```diff
@@ -0,0 +1,48 @@
+import { Instance, SnapshotOut, types } from "mobx-state-tree"
+
+import { IntroContentList } from "../../services/app-config"
+
+import { withEnvironment } from "../extensions"
+
+/**
+ * Intro content.
+ */
+export const IntroContentModel = types
+  .model("IntroContent")
+  .props({
+    /**
+     * Current content index
+     */
+    index: types.optional(types.number, 0),
+    /**
+     * Last update timestamp
+     */
+    lastUpdateTs: types.optional(types.number, 0),
+  })
+  .extend(withEnvironment)
+  .views(self => ({
+    get list() {
+      return self.getConfig<IntroContentList>(
+        "INTRO_CONTENT_SUPERLIKE_ID_LIST",
+        [],
+      )
+    },
+  }))
+  .views(self => ({
+    get current() {
+      return self.list[self.index]
+    },
+  }))
+  .actions(self => ({
+    increment() {
+      if (self.list.length > self.index) {
```

Comment:
Should be `self.index < self.list.length - 1`? Otherwise it is possible for `self.index` to reach `self.list.length`, then `self.list[self.index]` will return `undefined`